examples/admin-partitions/terraform/aws_ecs.tf

```diff
@@ -36,6 +36,22 @@ module "vpc" {
   single_nat_gateway   = true
   enable_dns_hostnames = true
   tags                 = var.tags
+  default_security_group_egress = [
+    {
+      from_port   = 0
+      to_port     = 0
+      protocol    = "-1"
+      cidr_blocks = "0.0.0.0/0"
+    }
+  ]
+  default_security_group_ingress = [
+    {
+      from_port   = 0
+      to_port     = 0
+      protocol    = "-1"
+      cidr_blocks = "0.0.0.0/0"
+    }
```

Comment:
The ingress rule should not allow traffic from any CIDR by default, to be safe. We can make this a self rule, so that only ingress from the same security group is allowed by default:

```
    {
      description = "Allow all"
      protocol = -1
      self = true
    }
```

But I think I'd prefer we remove these custom default security groups and try setting `manage_default_security_group = false` to see if that prevents Terraform from clobbering the AWS default rules. Up to you though.

Reply:
Making `manage_default_security_group` as `false` seems to be not altering the default security rules in the VPC. Going with that